lib/shopify_app/app_proxy_verification.rb

```diff
@@ -3,7 +3,7 @@ module AppProxyVerification
     extend ActiveSupport::Concern
 
     included do
-      skip_before_action :verify_authenticity_token
+      skip_before_action :verify_authenticity_token, raise: false
```

Comment:
Yeah, shopify_app passes the tests on 2.3.1 which is what shopify_api uses now. There were no changes necessary for Ruby 2.3.1. Just for Rails 5.0.